assets/src/util/calendar-utils.js

```diff
@@ -40,13 +50,18 @@ function businessHoursOrNearestHalfHour(
       ? `0${roundToNearest15(new Date()).getMinutes()}`
       : roundToNearest15(new Date()).getMinutes()
   }`;
-  if (today !== calendarDate) {
-    return businessStartHour;
+  if (
+    currentClosestHalfAnHourFormatted > adjustedBusinessHour &&
+    calendarDate === today
+  ) {
+    adjustedBusinessHour = currentClosestHalfAnHourFormatted;
   }
-  if (currentClosestHalfAnHourFormatted > businessStartHour) {
-    return currentClosestHalfAnHourFormatted;
+
+  if (returnMilliseconds) {
+    const timeParts = adjustedBusinessHour.split(":");
+    adjustedBusinessHour = +timeParts[0] * (60000 * 60) + +timeParts[1] * 60000;
```

Comment:
`+ +`  ?

Reply:
I guess to ensure that the number is positive. Yet i can't come up with an example where it would be negative. Removing excess "+"